src/webparts/todolist/components/ToDoList.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from "react";
+import styles from "./ToDoList.module.scss";
+import { IToDoListProps } from "./IToDoListProps";
+import { IItemListProps } from "./IItemListProps";
+import { escape } from "@microsoft/sp-lodash-subset";
+import { SPHttpClient } from "@microsoft/sp-http";
+import { Icon, List } from "office-ui-fabric-react";
+import * as strings from "ToDoListWebPartStrings";
+
+export interface ISPListItem {
+  Title: string;
+  Id: string;
+  Status: string;
+}
+
+export interface IListItemState {
+  items: ISPListItem[];
+  errorMessage: any;
+}
+
+class ItemList extends React.Component<IItemListProps, IListItemState> {
+  constructor(props: IItemListProps, state: IListItemState) {
+    super(props);
+
+    this.state = {
+      items: [],
+      errorMessage: null
+    };
+  }
+
+  private async _getListData(): Promise<ISPListItem[]> {
+    try {
+      const response = await this.props.spHttpClient.get(
+        "https://xq0nb.sharepoint.com/sites/TestSite/_api/web/lists/getByTitle('To do list')/items?$select=Title,Status",
```

Comment:
If you will try to use you webpart on another site - you stull get results from TestSite. It's not correct. You should use here the value from the context. Smth like that `this.context.pageContext.web.absoluteWebUrl`